scripts/v.unpack/v.unpack.py

```diff
@@ -77,7 +77,7 @@ def main():
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
         data_name = tar.getnames()[0]
-    except:
+    except (AttributeError, IndexError, tarfile.ReadError):
         grass.fatal(_("Pack file unreadable"))
```

Comment:
Same as the other extract code: Report the error to the user.

Reply:
I am guessing you are refering to ```scripts/r.in.srtm/r.in.srtm.py``` and changed to as following.
```python
    except (AttributeError, IndexError, tarfile.ReadError) as error:
        grass.fatal(_("Pack file unreadable: {error}").format(error=error))
```

Reply:
Did you push the change?